i18n/locales/source/dashboard/dsls.yml

```diff
@@ -2788,7 +2788,13 @@ en:
       coursef_powerofwords:
         description here: description here
         title: title
+      csd deeper learning timeline:
+        Deeper Learning Submissions Deadlines: Deeper Learning Submissions Deadlines
+        description here: description here
```

Comment:
This also seems weird but looks like has been a problem for a while. Many of these titles and descriptions aren't real. But then again we hide all `csd` and `csp` in Crowdin. So doesn't matter because translators won't see it, but will make your scripts run longer the bigger these files get.